conf_linux_public.py

```diff
@@ -107,6 +109,22 @@ def get_building_cmd(command, gcc_latest, enable_devtoolset):
        cmd=print_gcc_version(GCC_LATEST, ENABLE_DEVTOOLSET),
        verbose=True)
 
+#Build dependencies
+##Build LibVA
+action('autogen.sh', #TODO: Do we have to build with latest gcc?
```

Comment:
compiler version should be the same as for the product

Reply:
It is already implemented (see one line below)